setup.py

```diff
@@ -22,57 +22,57 @@
         #   tensorboard needs >=41
         #   50.0 is broken: https://github.com/pypa/setupatools/issues/2353
         "setuptools>=41.0.0,!=50.0",
-        "cached-property",
-        "click",  # used in scl
+        "cached-property==1.5.2",
+        "click==8.0.3",  # used in scl
         "gym==0.18.3",
-        "numpy",
-        "pandas",
-        "psutil",
+        "numpy==1.19.5",
+        "pandas==1.3.4",
+        "psutil==5.8.0",
         "pybullet==3.0.6",
-        "pynput",  # Used by HumanKeyboardAgent
-        "rich",
-        "rtree",  # Used by sumolib
-        "sh",
-        "shapely",
-        "sklearn",  # KDTree from sklearn is used by sumo lanepoints
-        "tableprint",
+        "pynput==1.7.4",  # Used by HumanKeyboardAgent
+        "rich==10.13.0",
+        "rtree==0.9.7",  # Used by sumolib
+        "sh==1.14.2",
+        "shapely==1.8.0",
+        "sklearn==0.0",  # KDTree from sklearn is used by sumo lanepoints
```

Comment:
This version seemed odd to me so I looked into it. It looks like the `PyPI` entry recommends to use `scikit-learn` instead.

Reply:
Changed `sklearn` to `scikit-learn`.